crawlerr/__init__.py

```
@@ -9,12 +9,22 @@
 __version__ = "0.1.0"
 __author__ = "Crawlerr Team"
 __description__ = "RAG-enabled web crawling MCP server"
 
-# Main exports
-from .server import mcp, main
+# Main exports - use lazy imports to avoid module conflicts
 from .config import settings
 
-__all__ = ["mcp", "main", "settings"]
\ No newline at end of file
+__all__ = ["settings", "get_mcp", "get_main"]
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)